http.c

```diff
@@ -2316,6 +2307,59 @@ evhttp_connection_base_bufferevent_new(struct event_base *base, struct evdns_bas
 	return (NULL);
 }
 
+struct evhttp_connection *
+evhttp_connection_unixsocket_new(struct event_base *base, struct bufferevent* bev, const char *unixsocket)
```

Comment:
I think this should not exist for windows, i.e. the whole function should be guarded with ifdef

Reply:
and I don't like the namings, maybe it can be done in the same manner as `evhttp_connection_base_bufferevent_new`, i.e. `evhttp_connection_base_bufferevent_unix_new` (not sure though, since this is pretty long)

Reply:
Done and Renamed to evhttp_connection_base_bufferevent_unix_new